### PR TITLE
Basic layout of the calculator app

### PR DESCRIPTION
**Pull request by Jolak5**
In this PR, I created the basic layout of the calculator app. 

I also fixed the linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,210 @@
...
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```
Comment (tresorsawasawa) [inline README.md]: - [ ] For clean code and as mentioned in the template itself, Kindly make sure you remove all the comments and instructions after you have finished to use the template

File: README.md [hunk 2]
```diff
@@ -1 +1,210 @@
...
+- You will be able to see a simple movie website with featured movies and series
+- Once you click on the comment button you should be able to see a pop-up modal with movie details and comment form
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```
Comment (tresorsawasawa) [inline README.md]: - [ ] To make your documentation more descriptive and professional, Kindly ensure that you customize your `# Getting Started` section with appropriate information related to your project and add all commands needed such as how to `clone your repo`, `access the cloned directory`, `install dependencies`, `run server(usage)`,... I suggest your refer this template to help you.

File: README.md [hunk 3]
```diff
@@ -1 +1,210 @@
...
+- has homepage with details about the a8pp
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jolak5.github.io/Math-Magicians/index.html)
+
```
Comment (tresorsawasawa) [inline README.md]: - [ ] To make your documentation more professional, kindly make sure you use a working link on your `[Live Demo...]` as the current one shows a `404` error page once clicked. If you need more information about deploying a react app you can refer this **[link](https://blog.logrocket.com/8-ways-deploy-react-app-free/)** as you will find at least **8 ways** on how to deploy a react app online.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Jolak5 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Jolak5,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**